platform/source/mbed_retarget.cpp

```diff
@@ -873,9 +1043,11 @@ extern "C" int PREFIX(_ensure)(FILEHANDLE fh)
     return fsync(fh);
 }
 #endif
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 
 extern "C" int fsync(int fildes)
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Same question: can you remove fsync entirely?

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/1712d7aa3c8623fa770dcad6516b100ee96f8cf3..51afce6721b21feff4019c27b53742b4e2d79872).